wagtail/images/models.py

```diff
@@ -182,9 +184,12 @@ def open_file(self):
                 image_file.close()
 
     @contextmanager
-    def get_willow_image(self):
-        with self.open_file() as image_file:
-            yield willow.Image.open(image_file)
+    def get_willow_image(self, existing_file: "File" = None):
```

Comment:
Issue: I'm not sure the `existing_file` is something which should be handled by `get_willow_image`. Nothing about that code branch uses `self`, or anything on the current instance. I think this would be better handled by the caller of `get_willow_image`.

Reply:
I see your point there. Happy to make this change.

Reply:
@RealOrangeOne This is addressed in [e494d13](https://github.com/wagtail/wagtail/pull/9550/commits/e494d13f3172df7ac89cf2afe0d9c4e45b03c152). Please let me know if you're happy with this fix.